service/provider/src/main/java/gov/va/vro/service/provider/services/ExamOrderSlackProcessor.java

```diff
@@ -0,0 +1,40 @@
+package gov.va.vro.service.provider.services;
+
+import gov.va.vro.model.claimmetrics.ExamOrdersInfo;
+import gov.va.vro.model.claimmetrics.response.ExamOrderInfoResponse;
+import gov.va.vro.model.event.AuditEvent;
+import gov.va.vro.service.provider.mas.MasException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+@Slf4j
+public class ExamOrderSlackProcessor implements Processor {
```

Comment:
Do we really need this processor? Why don't you prepare the body before entering to the camel route and just enter with that one?